src/interfaces.rs

```diff
@@ -70,9 +71,9 @@ pub fn dispatch(effect: *mut AEffect, opcode: i32, index: i32, value: isize, ptr
     let mut plugin = unsafe { (*effect).get_plugin() };
 
     // Copy a string into the `ptr` buffer
-    let copy_string = |string: &String, max: size_t| {
+    let copy_string = |string: &String, max: usize| {
         unsafe {
-            libc::strncpy(ptr as *mut c_char,
+            strncpy(ptr as *mut c_char,
```

Comment:
Let's leave this line as `libc::strncpy` to make it clear that we are not implementing our own.